packages/devextreme/testing/tests/DevExpress.ui.widgets.pivotGrid/store.remote.tests.js

```diff
@@ -1740,7 +1740,7 @@ QUnit.module('Expanding items', moduleConfig, () => {
                 { dataField: 'ShipRegion' },
                 { dataField: 'ShipCity' }
             ],
-            rowExpandedPaths: [['USA', 'AK']],
+            rowExpandedPaths: [['USA'], ['USA', 'AK']],
```

Comment:
`rowExpandedPaths: [['USA', 'AK']]` is as impossible expanded path in pivotgrid. To open 'AK', you have to first open 'US', so the expandedPaths must be at least this: `[['USA'], ['USA', 'AK']]`